deployments/infrastructure-target-aws.nix

```diff
@@ -33,11 +39,54 @@ rec {
     };
 
     deployment.ec2 = {
-      accessKeyId = IOHKaccessKeyId;
-
+      inherit accessKeyId;
+      elasticIPv4 = resources.elasticIPs.cardanod-ip;
       instanceType = mkForce "r3.2xlarge";
       ebsInitialRootDiskSize = mkForce 50;
       associatePublicIpAddress = true;
+      securityGroups = [
+        resources.ec2SecurityGroups."allow-all-ssh-${region}-${org}"
+      ];
+    };
+  };
+
+  resources = {
+    elasticIPs = {
+      hydra-ip    = { inherit region accessKeyId; };
+      cardanod-ip = { inherit region accessKeyId; };
+    };
+    ec2SecurityGroups = {
+      "allow-deployer-ssh-${region}-${org}" = {
```

Comment:
This security group is not needed because `allow-all-ssh` overrides it to all IPs

Reply:
We need to only allow the rest of infra to be accessed through the deployer only?

It was my intention, but I didn't carry it through..

Reply:
Then let's add a comment ` TODO: allow hydra and slaves to only have SSH access via deployer`

Reply:
@domenkozar, already implemented!

Reply:
Ah I see now :)